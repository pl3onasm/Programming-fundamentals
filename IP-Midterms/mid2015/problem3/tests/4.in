1000 5603
