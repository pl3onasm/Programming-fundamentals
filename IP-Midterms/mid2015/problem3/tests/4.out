1000*5603=1000+2000+32000+64000+128000+256000+1024000+4096000=5603000
